PERSONALITEMS.md

```diff
@@ -0,0 +1,143 @@
+# Personal Item Guidelines
+At Moffstation, players are able to create a cosmetic personal item for their characters.
+This is meant to be a fun way to express your character's personality, style, or story.
+The following guidelines are in place to ensure that personal items are appropriate for the game and do not interfere with or complicate gameplay.
+
+These goals and guidelines are similar in vain to Sector Umbra's personal items.
+
+## Core Item Guidelines
+1. **Personal items are limited in their functionality.**
+   1. They cannot be weapons, or have any combat-related functionality.
+      - Examples include reskinned cybersun pens, combat/survival knives, shovels, or any other item that could be used as a weapon.
+   2. They cannot be tools, or have any tool-related functionality.
+      - Examples include reskinned screwdrivers, wrenches, flashlights, or any other item that could be used as a tool.
+   3. They cannot be protective gear, or have any protective functionality.
+      - Examples include reskinned helmets, armor, or any other item that could be used as protective gear.
+      - Items like masks or winter coats are allowed as long as they remain mechanically the same as the original item it's based off of.
+      - This is relaxed for job-related personal items.
+2. **Personal items are limited in their visuals.**
+   1. They cannot be visually conflicting with any job clothing.
+      - People should be able to recognize you and not mistake you for other jobs or characters.
+      - For example, if you can be mistaken for a security officer or command member, you should not be wearing it.
+   2. They cannot mimic any emergency response (ERT) clothing.
+      - This includes any clothing that is similar to the ERT's clothing, such as the ERT's armor or helmets.
+   3. They cannot have any unrealistic or special effects.
+      - Examples include using the ninja stealth effect (`StealthComponent`), or jitter effect.
+      - Unshaded elements like flashing lights on the sprite are allowed.
+3. **Personal item descriptions are limited.**
+   1. Descriptions should be in-character and not break immersion.
+      - For example, OOC details about a long story are not allowed. These questions should be asked to the character in-game, not told through an item description.
+      - Try to always be _physically descriptive_ about the item.
+   2. Descriptions should not be excessively long or a wall of text.
+      - Try to make a very short, 1-2 sentence description of the item. Sometimes even one sentence about the item's appearance is good.
+      - Large descriptions of the item should be put under an `OuterDescription` component.
```

Comment:
![image](https://github.com/user-attachments/assets/4c31b6d0-381c-437e-8fcd-ccc9c22378a3)
?

Do you mean `DetailExaminable` like I've done on some of them?